maestro-orchestra-models/src/main/java/maestro/orchestra/Commands.kt

```diff
@@ -265,7 +265,8 @@ data class TapOnElementCommand(
 ) : Command {
 
     override fun description(): String {
-        return label ?: "${tapOnDescription(longPress, repeat)} on ${selector.description()}"
+        val optional = if (optional) "(Optional)" else ""
+        return label ?: "${tapOnDescription(longPress, repeat)} on $optional ${selector.description()}"
```

Comment:
```suggestion
        val optional = if (optional) "(Optional) " else ""
        return label ?: "${tapOnDescription(longPress, repeat)} on $optional${selector.description()}"
```

Reply:
Nit: to avoid having two spaces in a row when the command is not optional we can do it like this

Reply:
Makes sense! I'll do that